packages/components/tests/dummy/app/components/dummy-section/showcase.hbs

```diff
@@ -0,0 +1,11 @@
+<section data-test-percy aria-hidden="true">
+  <h3 class="dummy-h3" id={{or @id "showcase"}}><a href="#{{or @id 'showcase'}}" class="dummy-link-section">§</a>
```

Comment:
[issue] I think we should only hide the content for this section, because it's a [failure to hide an interactive element](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus/).

Reply:
But what if there are interactive elements inside the content of the section? How would we solve it?